README.md

```diff
@@ -68,6 +68,26 @@ An asterisk can be used as a wildcard character without having to enter the pack
 it:testOnly *SessionRepositorySpec
 ```
 
+
+## Run the application
+
+To update from Nexus and start all services from the RELEASE version instead of snapshot
+```
+sm2 --start IMPORT_ONE_STOP_SHOP_ALL
+```
+
+### To run the application locally execute the following:
+
+The service needs to run in testOnly mode in order to access the testOnly get-passcodes endpoint which will generate a passcode for email verification.
+```
+sm2 --stop IOSS_RETURNS_FRONTEND
+```
+and
+```
+sbt run -Dapplication.router=testOnlyDoNotUseInAppConf.Routes
+```
```

Comment:
Look like the same section is in the README above

Reply:
We both seemed to have added it :)